### ISSUE TITLE
Stuck on generating kubeconfig

### ISSUE THREAD

**Issue by myevit** [sighupio/permission-manager#112]
Stuck on generating kubeconfig
<img width="746" alt="image" src="https://user-images.githubusercontent.com/13375246/184698621-d1bdb9c9-e18a-444b-ad71-97359be97291.png">

**Comment by Scofields** [comment]
me too!

**Comment by ettoreciarcia** [comment]
I ran into the same problem

**Comment by loeken** [comment]
I noticed the same behaviour starting with kubernetes version 1.24. it still works in 1.23 for me, 1.25 also seems affected 
@myevit @Scofields @ettoreciarcia what version of kubernetes did you try installing it in?

**Comment by loeken** [comment]
when i create the user ( but not click on display kubeconfig ), it seems to create a permissionmanager user but no service accounts
❯ k get permissionmanagerusers
NAME                            AGE
permissionmanager.user.loeken   33s
❯ k get serviceaccounts --all-namespaces|grep loeken


after i click on view kubeconfig i get this error in the logs:
```
2022/10/03 10:13:43 timed out waiting for the condition
```
then the pod seems to crash and restart if i then click on view kubeconfig again i see this error in the logs

```
2022/10/03 10:17:46 Service Account not created: serviceaccounts "loeken" already exists
```
which i can confirm has been created:

```
❯ k get serviceaccounts --all-namespaces|grep loeken
permission-manager   loeken  
```

**Comment by dyipon** [comment]
same here with 1.25

**Comment by loeken** [comment]
Project dead?

**Comment by gajdosr** [comment]
Same here, figured out that from1.24 the ServiceAccountToken is not generating automatically.

Workaround:
--feature-gates=LegacyServiceAccountTokenNoAutoGeneration=false

Howto:
https://faun.pub/feature-gates-how-to-enable-kubernetes-alpha-features-fdb38dc3a9aa

**Comment by thegalah** [comment]
@gajdosr  this worked for me

**Comment by luigibarbato** [comment]
Hello guys! 🙋🏽‍♂️
First of all, thank you for all your contributions and especially for waiting. 😊

The problem you encountered is related to the changes that have occurred on the ServiceAccount since version v1.24: When a ServiceAccount is created, no more Secret is created automatically, which the Permission Manager used to say was a sure thing. Anyway, the problem will be fixed at the merge of the #109
